content/dstu2/related-person.md

```diff
@@ -0,0 +1,49 @@
+---
+title: RelatedPerson | DSTU 2 API
+---
+
+# RelatedPerson
+
+* TOC
+{:toc}
+
+## Terminology Bindings
+
+<%= terminology_table(:related_person, :dstu2) %>
+
+## Search
+
+Search for RelatedPersons that meet supplied query parameters:
+
+    GET /RelatedPerson?:parameters
+
+### Parameters
+
+ Name         | Required?                                                    | Type          | Description
+--------------|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------
+ `_id`        | No, if populated all other parameters are ignored            | [`token`]     | The logical resource id associated with the resource.
+ `identifier` | Yes, if neither `_id` nor `patient` are set                  | [`token`]     | A RelatedPerson identifier.
+ `patient`    | Yes, if neither `_id` nor `identifier` are set               | [`reference`] | A reference to a patient related which the RelatedPerson is associated. Example: `14067892`
```

Comment:
Could trim out some of the empty space in this table to keep the style consistent with other resources :)

```
 Name         | Required?                                         | Type          | Description
--------------|---------------------------------------------------|-------------------------------------------------------------------------------------------------------------
 `_id`        | No, if populated all other parameters are ignored | [`token`]     | The logical resource id associated with the resource.
 `identifier` | Yes, if neither `_id` nor `patient` are set       | [`token`]     | A RelatedPerson identifier.
 `patient`    | Yes, if neither `_id` nor `identifier` are set    | [`reference`] | A reference to a patient related which the RelatedPerson is associated. Example: `14067892`
```